chihacknight/decarbonize-my-state
# Fix React Errors & Improve About/Take Action Styling

## Overview

Made a few styling tweaks, particularly:

- Made headings bold by default
- Added some spacing to `h1` and `h2` elements

Also cleaned up several React errors where attributes where not camel case (e.g. `frameborder` needs to be `frameBorder` and `class` needs to be `className`). 

| Before | After |
| --------- | ------ |
| ![decarbmystate com_about (1)](https://user-images.githubusercontent.com/3187531/178175247-87477487-4719-4585-a8aa-02449771f4a3.png)  | ![localhost_8000_take-action](https://user-images.githubusercontent.com/3187531/178175248-2ff8da97-01da-4401-8ae3-6a50972a30ad.png) |
| ![decarbmystate com_about](https://user-images.githubusercontent.com/3187531/178175249-0076aa02-ee07-43c8-a525-1cab756e3567.png) | ![localhost_8000_illinois_](https://user-images.githubusercontent.com/3187531/178175251-eb62d7d7-ebd2-4306-b82e-0c29b4210b74.png) |

Also did some refactors resulting from this, removing extra `font-weight-bold` classes that are now no longer needed.

### Demo

See description.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Confirm the homepage and state details page are unchanged
* Confirm Take Action and About page look better and are more readable
